generator/src/main/java/com/scottlogic/deg/generator/generation/DecisionTreeDataGenerator.java

```diff
@@ -81,4 +92,12 @@ public DecisionTreeDataGenerator(
             .limit(maxRows)
             .peek(monitor::rowEmitted);
     }
+
+    private void visualiseTree(DecisionTree decisionTree, String title) {
+        try (Visualiser visualiser = visualiserFactory.create(VisualiserLevel.STANDARD, title)) {
+            visualiser.printTree(title, decisionTree);
+        } catch (Exception e) {
```

Comment:
is it possible to catch these in the visualisers?

Reply:
i will catch and rethrow as runtime exceptions in visualiser.printTree so callers don't need to do and trys.  likewise i have to remove ioexception from Autocloseable::close() method it needs to implement (that implemented so can be used in a  try-resource block) - and there will need to be a catch and re-throw as runtime as underlying string writer's close method will throw IOexception.  but again will do thsi lower down so callers like thsi class don't need to handle it